model/resetPassToken.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require("mongoose")
+const Schema = mongoose.Schema
+const resetPasswordSchema = new Schema({
+    byUser: { type: mongoose.Types.ObjectId, ref: 'users' },
+    token: { type: String, required: true },
+    expireIn: { type: Number, required: true, default: Date.now() + 3600000 } // 60 * 60 * 1000 = 3600000ms expires in 1 hour
+})
+
+const resetPaswordTokenModel = mongoose.model('resetPassToekns', resetPasswordSchema)
```

Comment:
For this one, as we previously used [kabel case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) for naming database model. So it should be `reset-pass-tokens`.